### PR TITLE
Feature/e UI 8397

### PR DESCRIPTION
**Pull request by MateiHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8397

### Change description ###
Selecting continue on the query details page on respond to query

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by johnbenjamin-hmcts — DISMISSED** [review]

**Review by Daniel-Lam — APPROVED** [review]
@MateiHMCTS - all looks OK; approved

**Review by DavidJayakumar — DISMISSED** [review]
Looks good

**Review by johnbenjamin-hmcts — DISMISSED** [review]

**Review by DavidJayakumar — APPROVED** [review]

**Review by johnbenjamin-hmcts — DISMISSED** [review]

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-check-your-answers/query-check-your-answers.component.html [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
   <div class="govuk-grid-column-two-thirds-from-desktop">
     <h1 class="govuk-heading-l">
+      <ng-container *ngIf="QueryCreateContext === queryItemTypeEnum.RESPOND; else genericQueryResponseMessage">
+        {{ 'Review query response details' | rpxTranslate }}
+      </ng-container>
+
+      <ng-template #genericQueryResponseMessage>
+        {{ 'Review query details' | rpxTranslate }}
+      </ng-template>
       {{ 'Review query details' | rpxTranslate }}
```
Comment (DavidJayakumar) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-check-your-answers/query-check-your-answers.component.html:11]: Why is {{ 'Review query details' | rpxTranslate }} one within template and one outside

**Review by DavidJayakumar — APPROVED** [review]
Looks good

**Review by johnbenjamin-hmcts — DISMISSED** [review]

**Review by DavidJayakumar — APPROVED** [review]

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Pull request merged**